deploy/operator.yaml

```diff
@@ -30,4 +30,4 @@ spec:
             - name: OPERATOR_NAME
               value: "nxiq-operator-certified"
             - name: RELATED_IMAGE_IQ
-              value: registry.connect.redhat.com/sonatype/nexus-iq-server:1.94.0-ubi-1
+              value: registry.connect.redhat.com/sonatype/nexus-iq-server:1.95.0-ubi-2
```

Comment:
doesn't match the version information in the container image above

Reply:
this is the ubi version of the app image. it had been deployed twice for whatever reason and latest is at *-2. the other version to which you are likely comparing this one is the version of the operator container.